lib/ts/recipe/session/sessionClass.ts

```diff
@@ -169,7 +176,7 @@ export default class Session implements SessionContainerInterface {
                     this.helpers.config,
                     this.reqResInfo.transferMethod,
                     this.reqResInfo.req,
-                    {}
+                    makeDefaultUserContextFromAPI(this.reqResInfo.req)
```

Comment:
user Context is from the input. If that's undefined, then call the makeDefaultUserContextFromAPI